base/process.jl

```diff
@@ -72,6 +72,10 @@ backticks, e.g.
 """
 Cmd
 
+hash(x::Cmd) = hash(x.exec, hash(x.env, hash(x.ignorestatus, hash(x.dir, hash(x.flags)))))
```

Comment:
hm looking at the other implementations of hash for Set, Dict, etc. they do the same. Good catch - I'll change this